### ISSUE TITLE
Bump/remove outdated `spyder-kernerls` `requirements.py` version validation and Spyder crashing

### ISSUE THREAD

**Issue by RolBurgHxgn** [spyder-ide/spyder#17005]
## Problem Description

Spyder crashes (without error message) when starting after updating to newest version 5.2.0 on Windows with the windows installer, when the python interpretor is set to a version which uses an outdated spyder kernel. 

### What steps reproduce the problem?

set Python interpreter in preferences to python version with outdated spyder kernel. Usually an error message appears in the console in such a situatino. But after the update to 5.2.0, Spyder just crashes without any error message. After Performing a Spyder --reset it started again. After Setting again to another python version, it crashed again.

### What is the expected output? What do you see instead?
Spyder wont start, but crashes. Even when using a wrong spyder-kernel version, the program should start up and display an error message, but it does not, just crashes

## Versions
Spyder 5.2.0
Python Version 3.9
Spyder Kernel Version 2.1.0
Windows 10

**Comment by RolBurgHxgn** [comment]
I think in requirements.py
```
def check_spyder_kernels():
    """Check spyder-kernel requirement."""
    try:
        import spyder_kernels
        required_ver = '2.0.0'
```
is outdated.

**Comment by dalthviz** [comment]
Hi @RolBurgHxgn thank you for the feedback! I think your issue regarding Spyder crashing due the spyder-kernels version is a duplicate of #16896 and a fix will be release with Spyder 5.2.1

> I think in requirements.py
> 
> def check_spyder_kernels():
>     """Check spyder-kernel requirement."""
>     try:
>         import spyder_kernels
>         required_ver = '2.0.0'
> 
> is outdated.

However regarding `requirements.py`  I think you are right, we should bump up the version check on that warning :+1:
